src/components/ListItem.jsx

```diff
@@ -1,5 +1,47 @@
+import { useEffect, useState } from 'react';
+import { updateItem } from '../api/firebase';
+
 import './ListItem.css';
 
-export function ListItem({ name }) {
-	return <li className="ListItem">{name}</li>;
+export function ListItem({ name, listToken, itemId, dateLastPurchased }) {
+	const [wasPurchased, setWasPurchased] = useState(false);
+
+	const handleCheck = () => {
+		updateItem(listToken, itemId);
+		setWasPurchased(true);
+	};
+
+	useEffect(() => {
+		const currentDate = new Date().getTime();
+
+		const dateLastPurchasedPlus24h = dateLastPurchased
+			? dateLastPurchased.toDate().getTime() + 60 * 60 * 24 * 1000
```

Comment:
What do you think about adding a variable here for 60 * 60 * 24 * 1000 (like one day in milliseconds) to make it easier to read?